calin/nz/_posts/2022-06-26-overview.md

```diff
@@ -0,0 +1,55 @@
+these hikes and skiing trips to be uneventful! So here we are, at the beginning of a
+new journey, one that feels exciting and gives me a good reason to challenge myself
+to write.
+
+Of course, everyone is asking why are we moving? The answer is ... complicated: it's
+been a while that we have been considering moving out of the US. We had an amazing
+run of 29 years here, and we are grateful for what this country has offered us:
+
+- **free education**, and actually even better -- we were paid to go to school,   learn, and earn our degrees.
+- **family life**: the opportunity to build a family; we have two loving kids who continue to teach and inspire us.
+- **job opportunities** in industries that have been growing exponentially and allowed us to live the "American dream": work hard and have a comfortable life.
+- **acceptance**: we never felt excluded or oppressed, even though we have  _funny accents_. It used to be that folks used that opportunity to ask   where are we from. It is also true that we _fit in_ -- smack in the middle of the white, middle class, educated class.
+- **natural beauty** to pursue our passion for the outdoors.
+
+Why are we leaving then?
+
+First and foremost, Google gave me an opportunity to continue the same work I'm
+doing now from New Zealand. The [Google NZ
+site](https://www.geekzone.co.nz/content.asp?contentid=24620){:target="_blank"} is a
+new office and with the experience in the past two years of working remotely,
+working in a timezone that is 3-5h away from California seems quite doable. We
+already have a couple of people working in that office and with the Google process
+for remote work, it's just too easy and appealing to pass on the opportunity. Thank
+you Google!
+
+New Zealand is a country of endless beauty, and as we looked around the world, it
+seems to be one of the countries that truly matches our lifestyle: small, outdoorsy,
+and yet ranking top on many issues that we care about. We do not care about gun
+ownership, we would rather abolish the capital punishment than the right to
+abortions, and we would rather have leadership than reality shows. New Zealand has a
+mature, adult government that takes care of its people. They have programs to reduce
+the inequality rather than always favouring the rich. Are these all an illusion
+because the grass is greener? We do not know for sure, but we'd rather find out. Are
+all these problems fixable? Indeed they are, but for that, we need the ability to
+have a civilized dialog. That does not seem to be possible anymore; Anca tried her
+hand to activism -- trying to convince people at least to go out and vote, and she
+found at most a handful of people willing to listen. The division runs deep and
+the political environment is only exacerbating the differences.
+
+So, just as 30 years ago we packed our bags and moved to the US to explore the world, here we are, packing our bags again and embarking on a new exploration.
```

Comment:
You could draw some parallels and talk about the last time you moved too, if those are relevant: what it was like then vs now, the differences in preparation/ideals/preconceptions when coming to this country from the last one vs moving to the next one, and what you've learned from the whole thing. To me that would be very interesting and a perspective not often seen.

Reply:
mmm, could that be the subject of an entirely new post? Once we figure out stuff here, it will be more meaningful to draw parallels.